Libs/DICOM/Widgets/ctkDICOMBrowser.cpp

```diff
@@ -422,7 +433,9 @@ ctkDICOMBrowser::~ctkDICOMBrowser()
 {
   Q_D(ctkDICOMBrowser);
 
+#if QT_VERSION >= QT_VERSION_CHECK(5,0,0)
```

Comment:
The check should be done based on enabled/disabled features. We should avoid features disappearing silently based on detected Qt version.